clients/unshaded/src/main/java/tachyon/client/lineage/TachyonLineageFileSystem.java

```diff
@@ -83,9 +83,9 @@ public FileOutStream getOutStream(TachyonURI path, OutStreamOptions options)
       return super.getOutStream(path, options);
     }
     if (fileId == -1) {
-      return new DummyFileOutputStream(fileId, options);
+      return new DummyFileOutputStream(path, options);
```

Comment:
Good point, do you mind if I do the renaming as a separate (final) step? Otherwise there could be a lot of noise.